docs/project/project.md

```diff
@@ -1,3 +1,3 @@
-# Project
+# Proje
 
-To create a successful project using TypeScript you need to understand the various project organization language features available. In this section we will cover "compilation context", declaration spaces and modules.
+TypeScript kullanarak başarılı bir proje oluşturmak için, mevcut olan çeşitli proje organizasyonu dil özelliklerini anlamanız gerekir. Bu bölümde, "derleme ortamı(compilation context)", deklarasyon alanları ve modülleri ele alacağız.
```

Comment:
İngilizce metinde kastedilen şeyin ne olduğundan emin olmamakla birlikte, benim çevirim şöyle bir şey olabilir:

TypeScript kullanarak başarılı bir proje oluşturmak için, dilin proje düzenini sağlayan çeşitli özelliklerini (yönlerini) kavramanız gerekli.